shared-spring/src/main/java/tds/shared/spring/interceptors/RestTemplateLoggingInterceptor.java

```diff
@@ -37,7 +37,7 @@ public ClientHttpResponse intercept(final HttpRequest request,
                                         final ClientHttpRequestExecution clientHttpRequestExecution) throws IOException {
 
         // get tracer id that used to associate all rest calls that are a child of this request
-        final String traceId = Optional.fromNullable(MDC.get(TRACER_ID_HEADER)).or(UUID.randomUUID().toString());
+        final String traceId = Optional.fromNullable(MDC.get(TRACER_ID_HEADER)).or(UUID.randomUUID().toString().replace("-", ""));
```

Comment:
I see how this fixes the problem.  Basically Spring's span class expects a max character length of 32 which with the dashes a UUID has 36 characters.  By removing the 4 `-` characters we get down to 32.